ubuntu22.04/precompiled/local-repo.sh

```diff
@@ -30,6 +36,16 @@ download_driver_package_deps () {
   apt-get download nvidia-fabricmanager-${DRIVER_BRANCH}=${DRIVER_VERSION}-1
   apt-get download libnvidia-nscq-${DRIVER_BRANCH}=${DRIVER_VERSION}-1
```

Comment:
Question: should we leverage the `download_apt_with_dep` function here as well?

Reply:
Yes, we should

Reply:
I will make that change